CONTRIBUTING.md

```diff
@@ -37,8 +37,8 @@ When you're not sure where your issue belongs, report it at [arduino/Arduino](ht
 - Search [existing pull requests and issues](https://github.com/arduino/Arduino/issues?q=) to be sure it hasn't already been reported. If you have additional information to provide about an existing issue then please comment on that issue. If you simply want to express your support then use the [Reactions feature](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments).
 - State the newest version of the Arduino IDE you have verified the issue with and which operating system you are using.
 - The issue title should be concise yet descriptive. Vague titles make it difficult to know the purpose of the issue when looking through the list of reports and may cause your issue to not be given proper attention.
-- Describe the issue and what behavior you were expecting. Post complete error messages using [Markdown code fencing](https://guides.github.com/features/mastering-markdown/#examples).
-- Provide a full set of steps necessary to reproduce the issue. Demonstration code should be complete, correct, and simplified to the minimum amount of code necessary to reproduce the issue. Please use [Markdown code fencing](https://guides.github.com/features/mastering-markdown/#examples) when posting code.
+- Describe the issue and what behavior you were expecting. Post complete error messages using [Markdown code fencing](https://guides.github.com/features/mastering-markdown/#examples) (ex: 3 back-ticks before and after code blocks: \`\`\`my code here\`\`\`).
```

Comment:
I feel this example will cause problems by making people think they can skip looking at the information at the link. The example works fine for a single line of output/code, but if you have multiple lines of output/code then each set of backticks need to be on their own line. This Markdown:
```
```foo
bar```
```
Renders as:
```foo
bar```

Reply:
OTOH, people will likely not read the link, so providing an example here directly might cause it to be seen better. But then the example might need to be a multi-line one perhaps?

Reply:
I didn't mean to imply I'm against providing an example. If we can teach people how to use code fencing from the contribution guide in a concise manner, that would be great. The link is a bit problematic because you need to click on the "Code" tab to find mention of code fencing.